City-of-Helsinki/drupal-helfi-platform
# UHF-1747 Webpack update

How to test: 
- Install platform: 
```
composer create-project City-of-Helsinki/drupal-helfi-platform:dev-UHF-1747_webpack_update hel-platform --no-interaction --repository https://repository.drupal.hel.ninja/
```
- Go to `hel-platform/public/themes/custom/hdbt_subtheme`
- Run `nvm use && npm i && npm run build` and check that there is no errors. Check also that dist folder gets compiled styles, js and icons.
